src/index.js

```diff
@@ -679,28 +679,38 @@ class NearProvider {
         try {
             let outcome;
             let val;
-            const { to, value, data } = txObj;
+            const { from, to, value, data } = txObj;
             if (value === undefined) {
                 val = new BN(0)
             } else {
                 const remove = utils.remove0x(value)
                 val = new BN(remove, 16)
             }
-            // value === undefined
-            //     ? new BN(0)
-            //     : new BN(utils.remove0x(value), 16);
 
-            // TODO: differentiate simple sends
-            const functionCallData = {
-                contractId: this.evm_contract,
-                methodName: 'deploy_code',
-                args: { 'bytecode': utils.remove0x(data) },
-                gas: GAS_AMOUNT,
-                amount: val
-            }
-
-            if (to === undefined) {
-                // Contract deployment.
+            if (data == undefined) {
```

Comment:
Change to `===`

Reply:
woops, didn't even realized I changed it.